src/utils.js

```diff
@@ -8,3 +8,31 @@ exports.findHighest = (...numbers) => {
     return highest;    
 };
 
+
+exports.findAverage = (...numbers) => {
+  let ave = 0;
```

Comment:
There is no reason to initialize this variable here. Also, lets write the variable name in full. Avoid `ave` instead of `average`, `numb` instead of `numbers`, etc

Reply:
noted